app/controllers/days_controller.rb

```diff
@@ -7,14 +7,15 @@ def nil.split *args
   end
   
   def index
+    @days = current_user.days
+
     if params[:days_between]
       start_split = params[:days_between].split(" - ").first
     @starts = Date.strptime(start_split, "%m/%d/%Y")
       end_split = params[:days_between].split(" - ").second
     @ends = Date.strptime(end_split, "%m/%d/%Y")
     else
     end
-    @days = Day.all
 
     @moodrating = Day.group(:moodrating)
 
```

Comment:
line 22 is why you can still see all the records in the system change this line to  @q = current_user.days.ransack(params[:q])

Reply:
Change is made.
Thanks for catching.